tests/testthat/output/trunc_mat/all--30.txt

```diff
@@ -1,10 +1,7 @@
 # A tibble: 3 x 9
-      a     b     c     d
+      a     b     c d    
```

Comment:
I think the principle is to right align numbers and left align anything else. Currently logical is right aligned, but should probably be left aligned?